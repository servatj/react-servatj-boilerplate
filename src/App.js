import React from 'react';

const App = () => {
  return <h1>Your react boilerplate is working fine!</h1>;
};

export default App;
