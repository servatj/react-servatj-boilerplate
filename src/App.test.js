import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';


test('Basic test App component', () => {
  const AppComponent = renderer.create(
    <App />
  )
  let tree = AppComponent.toJSON();
  expect(tree).toMatchSnapshot();
});
