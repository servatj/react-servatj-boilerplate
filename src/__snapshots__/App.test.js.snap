// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Basic test App component 1`] = `
<h1>
  Your react boilerplate is working fine!
</h1>
`;
